src/arrayMethodSort.js

```diff
@@ -4,8 +4,21 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = (a, b) => {
+    return a.toString() > b.toString();
+  }) {
+    for (let i = 0; i < this.length; i++) {
+      for (let k = i + 1; k < this.length; k++) {
+        if (compareFunction(this[i], this[k]) > 0) {
+          const valueI = this[i];
+
+          this[i] = this[k];
+          this[k] = valueI;
```

Comment:
you can use [destructuring assignment](https://javascript.info/destructuring-assignment#array-destructuring) for swapping values

Reply:
it`s a beautiful idea))